pysndfx/dsp.py

```diff
@@ -15,8 +15,9 @@
     FileBufferOutput
 
 
-class InvalidEffectParameter(Exception):
-    pass
+def mutually_exclusive(*args):
```

Comment:
This function is overly smart and hard to grok. Please use something from the standard library instead.

Reply:
I really thought about it, do you have any pointers for something from itertools or else that could do the job?

Reply:
```py
sum(arg is not None for arg in args) < 2
```

Reply:
Also, when it's just two arguments it's a lot easier to read inlined code where it's needed as:

```py
a = None
b = 2

if a is not None and b is not None:
    raise ValueError('Both a and b cannot be set at the same time.')
```